subprojects/enterprise-operations/src/main/java/org/gradle/operations/execution/ExecuteUnitOfWorkBuildOperationType.java

```diff
@@ -14,16 +14,16 @@
  * limitations under the License.
  */
 
-package org.gradle.operations.dependencies.transforms;
+package org.gradle.operations.execution;
 
 import org.gradle.internal.operations.BuildOperationType;
 
-public final class ExecuteTransformStepBuildOperationType implements BuildOperationType<ExecuteTransformStepBuildOperationType.Details, ExecuteTransformStepBuildOperationType.Result> {
+public final class ExecuteUnitOfWorkBuildOperationType implements BuildOperationType<ExecuteUnitOfWorkBuildOperationType.Details, ExecuteUnitOfWorkBuildOperationType.Result> {
+
     public interface Details {
         String getWorkspaceId();
```

Comment:
❓ Why "workspace" ID? Workspace seems like an implementation detail. Tasks for example don't even have workspaces currently. Why not call this simply "identity"?